src/qt/languages/fi-FI.po

```diff
@@ -38,7 +38,7 @@ msgid "R&emember size && position"
 msgstr "&Muista koko ja sijainti"
 
 msgid "Re&nderer"
-msgstr "&Renderöijä"
+msgstr "&Hahmonnin"
```

Comment:
While a correct term, this might be fairly unknown to common users.

Reply:
This is another case where it could go both ways. I personally think it's more important to have the "proper" term, even if it is somewhat unknown.